Fix search path to be on $PREFIX/pic30-nonfree, so it's easier to maintain the
Microchip's library (Just copy its include/ lib/ support/ into 
$PREFIX/pic30-nonfree)

--- c30_resource/src/c30/c30_flag_definitions.h.orig	2008-12-11 12:20:40.000000000 +0800
+++ c30_resource/src/c30/c30_flag_definitions.h	2008-12-11 12:30:03.000000000 +0800
@@ -72,18 +72,26 @@
  *   '/' is converted to the correct directory separator for the OS
  */
 #define MPLABC30_PIC30F_INC_PATH ".." DIR_SEPARATOR_STR \
+                                 ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "support" DIR_SEPARATOR_STR \
                                  "dsPIC30F" DIR_SEPARATOR_STR \
                                  "inc"
 #define MPLABC30_PIC33F_INC_PATH ".." DIR_SEPARATOR_STR \
+				 ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "support" DIR_SEPARATOR_STR \
                                  "dsPIC33F" DIR_SEPARATOR_STR \
                                  "inc"
 #define MPLABC30_PIC24H_INC_PATH ".." DIR_SEPARATOR_STR \
+                                 ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "support" DIR_SEPARATOR_STR \
                                  "PIC24H" DIR_SEPARATOR_STR \
                                  "inc"
 #define MPLABC30_PIC24F_INC_PATH ".." DIR_SEPARATOR_STR \
+                                 ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "support" DIR_SEPARATOR_STR \
                                  "PIC24F" DIR_SEPARATOR_STR \
                                  "inc"
@@ -94,46 +102,56 @@
  */
 #define MPLABC30_COMMON_INCLUDE_PATH ".." DIR_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
+                                     "pic30-nonfree" DIR_SEPARATOR_STR \
                                      "include" PATH_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
+                                     "pic30-nonfree" DIR_SEPARATOR_STR \
                                      "support" DIR_SEPARATOR_STR \
                                      "generic" DIR_SEPARATOR_STR \
                                      "h"
 #define MPLABC30_PIC24F_INCLUDE_PATH ".." DIR_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
+                                     "pic30-nonfree" DIR_SEPARATOR_STR \
                                      "support" DIR_SEPARATOR_STR \
                                      "PIC24F" DIR_SEPARATOR_STR \
                                      "h" PATH_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
+                                     "pic30-nonfree" DIR_SEPARATOR_STR \
                                      "support" DIR_SEPARATOR_STR \
                                      "peripheral_24F"
 #define MPLABC30_PIC24H_INCLUDE_PATH ".." DIR_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
+                                     "pic30-nonfree" DIR_SEPARATOR_STR \
                                      "support" DIR_SEPARATOR_STR \
                                      "PIC24H" DIR_SEPARATOR_STR \
                                      "h" PATH_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
+                                     "pic30-nonfree" DIR_SEPARATOR_STR \
                                      "support" DIR_SEPARATOR_STR \
                                      "peripheral_30F_24H_33F"
 #define MPLABC30_PIC30F_INCLUDE_PATH ".." DIR_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
+                                     "pic30-nonfree" DIR_SEPARATOR_STR \
                                      "support" DIR_SEPARATOR_STR \
                                      "dsPIC30F" DIR_SEPARATOR_STR \
                                      "h" PATH_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
+                                     "pic30-nonfree" DIR_SEPARATOR_STR \
                                      "support" DIR_SEPARATOR_STR \
                                      "peripheral_30F_24H_33F"
 #define MPLABC30_PIC33F_INCLUDE_PATH ".." DIR_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
+                                     "pic30-nonfree" DIR_SEPARATOR_STR \
                                      "support" DIR_SEPARATOR_STR \
                                      "dsPIC33F" DIR_SEPARATOR_STR \
                                      "h" PATH_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
                                      ".." DIR_SEPARATOR_STR \
+                                     "pic30-nonfree" DIR_SEPARATOR_STR \
                                      "support" DIR_SEPARATOR_STR \
                                      "peripheral_30F_24H_33F"
 /*
@@ -143,62 +161,76 @@
  */
 #define MPLABC30_COMMON_LIB_PATH ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "lib" PATH_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "support" DIR_SEPARATOR_STR \
                                  "generic" DIR_SEPARATOR_STR \
                                  "gld"
 #define MPLABC30_PIC24F_LIB_PATH ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "lib" DIR_SEPARATOR_STR \
                                  "PIC24F" PATH_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "support" DIR_SEPARATOR_STR \
                                  "PIC24F" DIR_SEPARATOR_STR \
                                  "gld" PATH_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "lib" DIR_SEPARATOR_STR \
                                  "peripheral_24F"
 #define MPLABC30_PIC24H_LIB_PATH ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "lib" DIR_SEPARATOR_STR \
                                  "PIC24H" PATH_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "support" DIR_SEPARATOR_STR \
                                  "PIC24H" DIR_SEPARATOR_STR \
                                  "gld" PATH_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "lib" DIR_SEPARATOR_STR \
                                  "peripheral_30F_24H_33F"
 #define MPLABC30_PIC30F_LIB_PATH ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "lib" DIR_SEPARATOR_STR \
                                  "dsPIC30F" PATH_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "support" DIR_SEPARATOR_STR \
                                  "dsPIC30F" DIR_SEPARATOR_STR \
                                  "gld" PATH_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "lib" DIR_SEPARATOR_STR \
                                  "peripheral_30F_24H_33F"
 #define MPLABC30_PIC33F_LIB_PATH ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "lib" DIR_SEPARATOR_STR \
                                  "dsPIC33F" PATH_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "support" DIR_SEPARATOR_STR \
                                  "dsPIC33F" DIR_SEPARATOR_STR \
                                  "gld" PATH_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
                                  ".." DIR_SEPARATOR_STR \
+                                 "pic30-nonfree" DIR_SEPARATOR_STR \
                                  "lib" DIR_SEPARATOR_STR \
                                  "peripheral_30F_24H_33F"
 
